crates/router/src/connector/rapyd/transformers.rs

```diff
@@ -435,51 +361,75 @@ impl TryFrom<&types::PaymentsCaptureRouterData> for CaptureRequest {
     }
 }
 
-impl TryFrom<types::PaymentsCaptureResponseRouterData<RapydPaymentsResponse>>
-    for types::PaymentsCaptureRouterData
+impl<F, T>
+    TryFrom<types::ResponseRouterData<F, RapydPaymentsResponse, T, types::PaymentsResponseData>>
```

Comment:
This is a generic response handler, and redirection handling also here. For all flows redirection handling won't be there in core. That won't be problem?

Reply:
No for all other flow this data will be null, And i have tested it for all the flow that we currently integrated as of today